index.html

```diff
@@ -20,377 +20,273 @@
+                no accounts or sign-ups required. has been the industry's standard
+              </p>
+                <div class="tags-2">
+                  <ul>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                  </ul>
+                </div>
+   
+            </div>
           </div>
-          <div class="sixth-tag3">
-           <span>Ruby</span>
+  
+          <div class="third-card">
+            <div class="third-block">
+              <h1>Website Protfolio </h1>
+              <p>A daily selection of privately personalized reads; 
+                no accounts or sign-ups required. has been the industry's standard
+              </p>
+                <div class="tags-3">
+                  <ul>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                  </ul>
+                </div>
+            </div>
           </div>
-         </div>
-
-        <div class="action">
-          <h3>See Project</h3>
         </div>
-
-
-
-
-        
-
-      </section>
-      <section class="projects">
-
-        
-        <h1>Profesional Art Printing Data</h1>
-        <small>A daily selection of privately personalized reads; 
-          no accounts or sign-ups required. 
-          has been the industry's standard
-        </small>
-        <div class="four-tags">
-          <div class="four-tag">
-           <span>Html</span>
+  
+        <div class="card-2">
+          <div class="fourth-card">
+            <div class="fourth-block">
+              <h1>Profesional Art Printing Data More</h1>
+              <p>A daily selection of privately personalized reads; 
+                no accounts or sign-ups required. has been the industry's standard
+              </p>
+                <div class="tags-4">
+                  <ul>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                  </ul>
+                </div>
+            </div>
           </div>
-          <div class="four-tag2">
-           <span>bootstrap</span>
+          <div class="fifth-card">
+            <div class="fifth-block">
+              <h1>Profesional Art Printing Data More</h1>
+              <p>A daily selection of privately personalized reads; 
+                no accounts or sign-ups required. has been the industry's standard
+              </p>
+                <div class="tags-5">
+                  <ul>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                  </ul>
+                </div>
+            </div>
           </div>
-          <div class="four-tag3">
-           <span>Ruby</span>
+          <div class="sixth-card">
+            <div class="sixth-block">
+              <h1>Website Portfolio</h1>
+              <p>A daily selection of privately personalized reads; 
```

Comment:
- [ ] [Optional] I would suggest you please use `hx` tag here like `h2, h3,h4` because only we use the `h1` tag per page and it represents the main heading of the page.